### ISSUE TITLE
[Validator] Sends `Cannot read properties of undefined (reading 'match')` when there are missing fields.

### ISSUE THREAD

**Issue by unworthyEnzyme** [honojs/hono#705]
I have a schema like this:
```ts
{
    username: v
      .json("username")
      .isRequired()
      .isLength({ min: 1 })
      .message("username is required and cannot be empty"),
    password: v
      .json("password")
      .isRequired()
      .isLength({ min: 1 })
      .message("password is required and cannot be empty"),
  }
```
When i send a json body like this `{"username": "memo"}`, validator send `Cannot read properties of undefined (reading 'match')`.
I use the version `2.5.9` with nodejs.

**Comment by yusukebe** [comment]
Hi @unworthyEnzyme !

This is a bug! I've fixed it and released the new version `v2.5.10` including the change. Please upgrade Hono.

Thank you!

**Comment by unworthyEnzyme** [comment]
Thank you for a fast response.
